### PR TITLE
Fix comment typo regarding use of `packages:` and relationship to `package_update:`

### PR DESCRIPTION
**Pull request by magnetikonline**
## Proposed Commit Message

Fix comment typo regarding use of `packages:` and relationship to `package_update:`

```
Found a small typo in the online documentation.
```

## Additional Context

N/A

## Test Steps

N/A

## Checklist:

 - [x] My code follows the process laid out in [the documentation](https://cloudinit.readthedocs.io/en/latest/topics/contributing.html)
 - [ ] I have updated or added any unit tests accordingly
 - [x] I have updated or added any documentation accordingly

### CODE REVIEW

**Review by aciba90 — CHANGES_REQUESTED** [review]
Thanks, @magnetikonline, for making `cloud-init` better.

Could you please sign the CLA and add yourself to [tools/.github-cla-signers](https://github.com/canonical/cloud-init/blob/main/tools/.github-cla-signers) ?

More info: https://cloudinit.readthedocs.io/en/latest/topics/contributing.html#summary

File: tools/.github-cla-signers [hunk 1]
```diff
@@ -62,6 +62,7 @@ lkundrak
 lucasmoura
 lucendio
 lungj
+magnetikonline
```
Comment (magnetikonline) [inline tools/.github-cla-signers:65]: Username added @aciba90 👍

**Comment by magnetikonline** [comment]
> Could you please sign the CLA and add yourself to [tools/.github-cla-signers](https://github.com/canonical/cloud-init/blob/main/tools/.github-cla-signers) ?

Thanks @aciba90 - all done 👍

**Review by aciba90 — APPROVED** [review]
LGTM!

**Comment by magnetikonline** [comment]
Thanks @aciba90

**Review by TheRealFalcon — APPROVED** [review]
Thanks for the contribution, @magnetikonline !

**Comment by magnetikonline** [comment]
Not a problem @TheRealFalcon - just been getting into another great Canonical project in Multipass - and started authoring `cloud-init.yml` files. 👍

**Pull request merged**